xlr/sdk/README.md

```diff
@@ -1,10 +1,19 @@
 # @player-tools/xlr-sdk
 
-Unified interface for working with XLRs. Abstracts away needing to know how they are packaged or where they are sourced from. Current operations supported: 
+Unified interface for working with XLRs. Abstracts away needing to know how they
+are packaged or where they are sourced from. Current operations supported:
 
 - Load
 - Export
   - Typescript
 - Validate
 
-Also exposes functionality to create your own XLR Registry to manage how the SDK stores loaded types. 
\ No newline at end of file
+Also exposes functionality to create your own XLR Registry to manage how the SDK
+stores loaded types.
+
+## Validation
+
+To run XLR validation, `yarn run player dsl compile` which will display any
+relevant errors.
+
+For additional debugging information, `yarn run player dsl compile -v info`.
```

Comment:
_Technically_ validation is run via the `json validate` command but it is also called as part of DSL comilation